internal/querynode/flow_graph_query_node.go

```diff
@@ -204,7 +204,7 @@ func (q *queryNodeFlowGraph) newDmInputNode(ctx context.Context, factory msgstre
 	maxParallelism := Params.QueryNodeCfg.FlowGraphMaxParallelism.GetAsInt32()
 	name := fmt.Sprintf("dmInputNode-query-%d-%s", collectionID, vchannel)
 	node := flowgraph.NewInputNode(insertStream, name, maxQueueLength, maxParallelism, typeutil.QueryNodeRole,
-		paramtable.GetNodeID(), collectionID, dataType)
+		paramtable.GetNodeID(), collectionID, dataType, nil)
```

Comment:
When the input node is closed, the `syncChan` will be closed, the chan can't be nil.